### ISSUE TITLE
Node v14+ support and fix for TypeScript and EventEmitter

### ISSUE THREAD

**Issue by fidian** [fluent/fluent-logger-forward-node#25]
Node v14 is the active LTS and it switches to "maintenance" tomorrow, making Node v16 the current LTS as of Tuesday. It would be great to be able to upgrade the project so it builds without errors. When I try, I get the following:

```
node_modules/@fluent-org/logger/build/src/server.d.ts:63:43 - error TS2507: Type 'typeof EventEmitter' is not a constructor function type.

63 export declare class FluentServer extends EventEmitter {
                                             ~~~~~~~~~~~~

node_modules/@fluent-org/logger/build/src/socket.d.ts:164:43 - error TS2507: Type 'typeof EventEmitter' is not a constructor function type.

164 export declare class FluentSocket extends EventEmitter {
                                              ~~~~~~~~~~~~


Found 2 errors.
```

From what I see at [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41353#issuecomment-579381008), the upgrade at 13.4 or 13.5 caused issues. [Geckos.io](https://github.com/geckosio/geckos.io/issues/22) has a simple fix, as does [cheap-watch](https://githubmemory.com/repo/Conduitry/cheap-watch/issues/4). It basically means you have to change the import line.

```
// Old
import * as EventEmitter from "events";

// New
import { EventEmitter } from 'events';
```

**Comment by jamiees2** [comment]
Sorry for taking so long on this, I finally found some time, and addressed this in the recently cut v1.0.4 :) Let me know if that works for you!

**Comment by tance77** [comment]
This still seems to be an issue in node 16. Was this fixed?
